filament/include/filament/RenderableManager.h

```diff
@@ -340,6 +342,74 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
         Builder& skinning(size_t boneCount, Bone const* bones) noexcept; //!< \overload
         Builder& skinning(size_t boneCount) noexcept; //!< \overload
 
+        /**
+         * Define bone indices and weights for vertex skinning. The first pair value defines index of the bone and the
+         * second value is the bone weight. The pairs substitute \c BONE_INDICES and the \c BONE_WEIGHTS
+         * defined in the VertexBuffer. Both ways of indices and weights definition must not be combined in one primitive.
+         * Number of pairs is not limited to 4 bones per vertex.
+         * All bone weights of one vertex should sum to one. Otherwise they will be normalized.
+         * Data must be rectangular and number of pairs must be same for all vertices of this primitive.
+         * The data is arranged sequentially, all bone pairs for the first vertex, then for the second vertex, and so on.
+         *
+         * @param primitiveIndex zero-based index of the primitive, must be less than the primitive count passed to Builder constructor
+         * @param indicesAndWeights pairs of bone index and bone weight for all vertices sequentially,
+         * @param count number of all pairs, must be a multiple of vertexCount of the primitive
+         * @param offset specifies where in the indicesAndWeights buffer to start reading (expressed as a number of pairs)
+         *
+         * @return Builder reference for chaining calls.
+         */
+        Builder& boneIndicesAndWeights(size_t primitiveIndex,math::float2 const* indicesAndWeights, size_t count, size_t offset = 0);
```

Comment:
The `offset` parameter is not needed as it doesn't add value to the API. The same behavior can be achieved using `indicesAndWeights + offset`.

Reply:
@pixelflinger Ok, I will remove this parameter